packages/fx-core/templates/plugins/resource/identity/bicep/param.template.bicep

```diff
@@ -1,3 +1,3 @@
 
-param identity_managedIdentityName string = '${resourceBaseName}-managedIdentity'
+param identity_managedIdentityName string = 'managedIdentity-${uniqueString('${resourceBaseName}${utcNow()}')}'
```

Comment:
will we get different identity name when provision same project multiple times?

Reply:
We added time utcNow() in uniqueString. The name will be different when time changes.